src/System.Threading.Tasks.Extensions/ref/Configurations.props

```diff
@@ -7,6 +7,7 @@
     </PackageConfigurations>
     <BuildConfigurations>
       $(PackageConfigurations);
+      _netcoreapp2.1;
```

Comment:
Why is with `underscore` prefix?

Reply:
The underscore makes this a placeholder configuration. So while building all configurations if it sees that configuration it will skip building anything for that configuration. This is needed in this case to prevent the netstandard configuration from being bin-placed into the netcoreapp2.1 ref folder during the ref build. Because for netcoreapp2.1 some of the APIs were moved down into System.Runtime and conflict with the APIs in the netstandard ref of this library.